packages/core/src/awsService/appBuilder/serverlessLand/metadataManager.ts

```diff
@@ -62,9 +64,18 @@ export class MetadataManager {
                 const metadataContent = nodefs.readFileSync(metadataPath, { encoding: 'utf-8' })
                 const parseMetadata = JSON.parse(metadataContent) as ProjectMetadata
                 this.metadata = parseMetadata
+                telemetry.record({
+                    action: 'serverlessland_metadataLoaded',
+                    result: 'Succeeded',
```

Comment:
Since this is all wrapped in `telemetry.lambda_createServerlessLandProject.run`, you don't need to add a result since if no errors are thrown it will automatically populate `result: Succeeded`